docs/topics/request-response.rst

```diff
@@ -32,11 +32,22 @@ Request objects
     :type url: str
 
     :param callback: the function that will be called with the response of this
-       request (once it's downloaded) as its first parameter. For more information
-       see :ref:`topics-request-response-ref-request-callback-arguments` below.
-       If a Request doesn't specify a callback, the spider's
-       :meth:`~scrapy.Spider.parse` method will be used.
-       Note that if exceptions are raised during processing, errback is called instead.
+       request (once it's downloaded) as its first parameter.
+
+       In addition to a function, the following values are supported:
+
+       -   ``None`` (default), which indicates that the spider's
+           :meth:`~scrapy.Spider.parse` method must be used.
+
+       -   :py:data:`scrapy.http.request.NO_CALLBACK`
+
+            .. autodata:: scrapy.http.request.NO_CALLBACK
```

Comment:
There might be some other places to change in the docs, see e.g. https://docs.scrapy.org/en/latest/topics/item-pipeline.html#screenshotpipeline